emulators/dosbox-x/Portfile

```diff
@@ -7,12 +7,12 @@ PortGroup           app 1.0
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           muniversal 1.0
 
-github.setup        joncampbell123 dosbox-x 2024.03.01 dosbox-x-v
+github.setup        joncampbell123 dosbox-x 2024.07.01 dosbox-x-v
 revision            1
```

Comment:
`revision` needs to reset to 0 when `version` increases.

Reply:
Hi @herbygillot, I have fixed this.  Thanks.